configuration.rst

```diff
@@ -410,6 +410,93 @@ In reality, each environment differs only somewhat from others. This means that
 all environments share a large base of common configuration, which is put in
 files directly in the ``config/packages/`` directory.
 
+.. tip::
+
+    Starting from Symfony 5.3, you can also define options for different environments in a single file :
```

Comment:
```suggestion
    Starting from Symfony 5.3, you can also define options for different environments in a single file :
```

You can also define options for different environments in a single configuration file.

.. versionadded:: 5.3

     The ability to defined different environments in a single file was introduced in Symfony 5.3.

Reply:
This is better, indeed. :) Thanks!